loop_rate : 10.0
odometry_robot_name: "iris"
# imu_topic_name : "/scout_1/imu"
odometry_dr_topic_name : "/mavros/local_position/odom"
odometry_truth_in_topic_name: "/gazebo/model_states"
odometry_truth_out_topic_name: "/odometry/truth"
odometry_err_topic_name: "/odometry/error"
odometry_frame_id : "iris/odom"
odometry_child_frame_id : "iris/base_link"
